lib/package.gi

```diff
@@ -1898,9 +1901,6 @@ InstallGlobalFunction( AutoloadPackages, function()
     fi;
     LogPackageLoadingMessage( PACKAGE_DEBUG, msg, "GAP" );
```

Comment:
I assume you move the `SetInfoLevel` so that the above `LogPackageLoadingMessage` is shown? That makes sense to me

Reply:
Have you looked at the commit message bodies? If yes I'm of course happy to elaborate, but from your questions I have the feeling that they contain the details you are looking for, that's why I'm asking,

Reply:
Sorry I missed the bit about the commit messages